src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DataGridViewAccessibleObjectTests.cs

```diff
@@ -1629,5 +1629,34 @@ public void DataGridViewAccessibleObject_RowCount_ReturnsExpected_IfThreeColumnH
             Assert.Equal(0, dataGridView.AccessibilityObject.RowCount);
             Assert.False(dataGridView.IsHandleCreated);
         }
+
+        [WinFormsFact]
+        public void DataGridViewAccessibleObject_ItemStatusProperty_IsExpected_ForNonSortedDGV()
+        {
+            using DataGridView dataGridView = new();
+            dataGridView.Columns.Add(new DataGridViewTextBoxColumn());
```

Comment:
@kirsan31 pointed in https://github.com/dotnet/winforms/issues/6859 that `DataGridViewTextBoxColumn` is a disposable object, and DGV is very liberal in handling its elements lifecycle.
```suggestion
            using DataGridViewTextBoxColumn column = new();
            dataGridView.Columns.Add(column);
```

Reply:
Yes, thank you! I've implemented your variant.